etc/skydive.yml.default

```diff
@@ -104,6 +104,9 @@ agent:
     # stats_update: 1
   metadata:
     info: This is compute node
+  snort:
+    binary: /home/nplanel/Downloads/snort-3.0.0-a4/b/src/snort
```

Comment:
This is unused in the code, is this normal ?

Reply:
Yes I forgot to removed it